practice_2/AnimalsApp/AnimalsListModule/Presenter/AnimalsListPresenter.swift

```diff
@@ -0,0 +1,171 @@
+import Foundation
+
+let cacheExpired = 86400  // one day
+let maxNumberOfRequests = 30
+
+protocol AnimalsListViewProtocol: AnyObject {
```

Comment:
avoid using AnyObject

Reply:
At the moment, I cannot offer any other solutions. Honestly, I thought.
Apparently, we need to go much deeper.